app/views/claims/bank_details.html.erb

```diff
@@ -19,22 +20,22 @@
             <%= form.label :bank_sort_code, "Sort code", class: "govuk-label" %>
             <span  id="sort-code-hint" class="govuk-hint">For example: 44 00 26</span>
             <%= form.text_field :bank_sort_code,
-              class: "govuk-input govuk-!-width-one-quarter",
+              class: css_classes_for_input(current_claim, :bank_sort_code, "govuk-!-width-one-quarter"),
```

Comment:
Do we also need a `<%= errors_tag current_claim, :bank_sort_code %>` to go with this text field?

Reply:
Yes, thank you.